.github/workflows/python-dev.yml

```
@@ -75,7 +75,7 @@ jobs:
       run: |
         python --version
         python -m pip install --upgrade pip setuptools wheel
-        python -m pip install git+https://github.com/numpy/numpy.git
+        python -m pip install -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
```

```suggestion
        python -m pip install numpy==1.23.3
```

They have wheels for mac, windows, and linux:
https://pypi.org/project/numpy/1.23.3/#files

---

Hm, it might be worth it to stay with numpy main, until Python 3.11 is out, just in case compat issues arise between numpy and Python 3.11.